Makefile

```diff
@@ -0,0 +1,19 @@
+PROJECT  := infobloxopen/buildtool
+
+# The list of buildtool versions we are going to build,
+# make all versions as phony rules, so we will rebuild
+# them every time.
+VERSIONS := 0.0.1
```

Comment:
Think it makes sense to have the buildtool with a version of "latest", in addition to the internal versioning.

Also, this buildtool is quite simple and straightforward (no major/minor semantic versioning) so it might make sense to just name the version in a linear way as
`v1`, `v2`, `v...`